test/integration/targets/azure_rm_roledefinition/aliases

```diff
@@ -1,3 +1,4 @@
 cloud/azure
 shippable/azure/group1
 destructive
+disabled
```

Comment:
Use `unsupported` instead of `disabled`. The use of `disabled` is for tests which are temporarily broken, while `unsupported` is for tests we cannot run in CI.

Reply:
fixed